pkg/collector/corechecks/net/networkv2/network.go

```diff
@@ -295,24 +305,55 @@ func handleEthtoolStats(sender sender.Sender, interfaceIO net.IOCountersStat) er
 		}
 	}
 
-	processedMap := getEthtoolMetrics(driverName, statsMap)
-	for extraTag, keyValuePairing := range processedMap {
+	if collectEnaMetrics {
+		enaMetrics := getEnaMetrics(driverName, statsMap)
 		tags := []string{
-			"interface:" + interfaceIO.Name,
+			"device:" + interfaceIO.Name,
 			"driver_name:" + driverName,
 			"driver_version:" + driverVersion,
-			extraTag,
 		}
 
-		for metricName, metricValue := range keyValuePairing {
+		count := 0
+		for metricName, metricValue := range enaMetrics {
 			metricName := fmt.Sprintf("system.net.%s", metricName)
```

Comment:
the full prefix here is eg. `system.net.aws.ec2.bw_in_allowance_exceeded` for the ENA metrics